Form/Type/DateRangeType.php

```diff
@@ -20,15 +20,24 @@
 class DateRangeType extends AbstractType
 {
     /**
-     * @var TranslatorInterface
+     * @var TranslatorInterface|null
+     *
+     * @deprecated Translator property is deprecated since version 3.1, to be removed in 4.0.
      */
     protected $translator;
 
     /**
-     * @param TranslatorInterface $translator
+     * @param TranslatorInterface|null $translator
+     *
+     * @deprecated Translator dependency is deprecated since version 3.1, to be removed in 4.0.
```

Comment:
@core I don't agree with you. In code should be exactly said from which version the code is deprecated.
If the PR is merged later, just the 3.1 will be fixed.